### ISSUE TITLE
[Android] Browser window zooms in when some specific elements gets focus

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor5#1070]
## Is this a bug report or feature request? (choose one)

🐞 Bug report

## 💻 Version of CKEditor

Latest master (happens also on `v10.0.1`)

## 📋 Steps to reproduce

1. Go to https://ckeditor5.github.io/ on Android Chrome.
1. Focus a link.
1. Click on edit button inside link toolbar.

Also on the gif below it is visible that after inserting link, when whole editable gets focused, browser window also zooms in.

![chrome android zoom](https://user-images.githubusercontent.com/1061942/41591662-a57718d0-73ba-11e8-931d-69c7dcd97819.gif)
_The gif is from debug view but all clicking was performed on Android device_.

## ✅ Expected result

Edit input is opened, no change in a window zoom.
I'm not sure what should be the best behaviour but this zooming is rather annoying and makes editor harder to use.

## ❎ Actual result

Edit input is opened, the window zooms in.

## 📃 Other details that might be useful

Tested on Chrome `67.0.3396.87` on Android `4.4.4`.

IMHO zooming on focus seems like some native Android/Chrome behaviour.

**Comment by f1ames** [comment]
> First of all, the page on which CKEditor 5 is used needs to be mobile friendly. Our docs are not that friendly yet. So, please retest it on a page which has a proper font sizes, width, etc.

Checked on  https://ckeditor5.github.io which is more mobile friendly, but still zooming seems quite inconvenient.

> Second, this is a native behaviour. And we cannot block it, AFAIK. In fact, we shouldn't. The only thing we can do is to educate others (and, apparently ourselves too) that the page must be styled correctly.

I agree. I wonder if we would be able to create such styled page that zooming experience will be smooth. Maybe we should put it in a docs too?

**Comment by Reinmar** [comment]
> I agree. I wonder if we would be able to create such styled page that zooming experience will be smooth. Maybe we should put it in a docs too?

Yeah, definitely. That should be our goal too to define such rules, when working on mobile support.
